elenacaseyroby/YoutubeMusicLib
# Split yoututbe title into title and artist and store in db

# What
- Splits Youtube Title into Artist and Title and stores in db.
- Escapes quotes so user can now play and store videos with quotes in the title and description.  (There are a few exceptions to this, but it is a low priority bug fix since it works most of the time.)
# Why

So we can gather more metadata and pull metadata from other APIs
# Test
1. Go to localhost:5000 and play video for over 2 secs
2. Go to listens page and check that play is stored
3. Make changes to artist, title, album and make sure changes hold
4. Merge
